run.py

```diff
@@ -362,20 +362,24 @@ def run_assembly(asm_test, iss_yaml, isa, mabi, iss_opts, output_dir, setting_di
   """Run a directed assembly test with spike
 
   Args:
-    asm_tset    : Assembly test file
+    asm_test    : Assembly test file
     iss_yaml    : ISS configuration file in YAML format
     isa         : ISA variant passed to the ISS
     mabi        : MABI variant passed to GCC
     iss_opts    : Instruction set simulators
+    output_dir  : Output directory of compiled test files
     setting_dir : Generator setting directory
   """
   cwd = os.path.dirname(os.path.realpath(__file__))
   asm = re.sub(r"^.*\/", "", asm_test)
   asm = re.sub(r"\.S$", "", asm)
   report = ("%s/iss_regr.log" % output_dir).rstrip()
-  elf = ("%s/%s.o" % (output_dir, asm))
-  binary = ("%s/%s.bin" % (output_dir, asm))
+  prefix = ("%s/directed_asm_tests/%s"  % (output_dir, asm))
+  elf = prefix + ".o"
+  binary = prefix + ".bin"
   iss_list = iss_opts.split(",")
+  run_cmd("mkdir -p %s/directed_asm_tests" % output_dir)
+  run_cmd("cp %s %s/directed_asm_tests/" % (asm_test, output_dir))
```

Comment:
One potential issue is that these hand coded tests may have dependencies to other include files, copying this file to a different directory could break this. I'd prefer to keep the hand-coded tests in the original directory, only redirect .bin/.o to directed_asm_tests directory

Reply:
hmm, okay, I'll modify this accordingly

Reply:
Done.